src/components/Person/Person.jsx

```diff
@@ -1 +1,50 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Person = ({
+  person: {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  },
+}) => {
+  const partner = sex === 'm'
+    ? ('wife')
+    : ('husband');
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+
+      {age && (
+        <p className="Person__age">
+          {`I am ${age}`}
+        </p>
+      )}
+
+      <p className="Person__partner">
+        {isMarried
+          ? (`${partnerName} is my ${partner}`)
+          : ('I am not married')
+        }
+      </p>
```

Comment:
```suggestion
      <p className="Person__partner">
        {isMarried
          ? (`${partnerName} is my ${sex === m ? 'wife' : 'husband'}`)
          : ('I am not married')
        }
      </p>
```
in this way u don't need to use partner statement)

Reply:
implement ternary operator inside the ternary operator makes code less readability.